controllers/past-grants/views/index.njk

```diff
@@ -13,7 +13,8 @@
             queryParams: {{ queryParams | dump | safe }},
             totalResults: {{ meta.totalResults }},
             totalAwarded: {{ meta.totalAwarded}},
-            sort: {{ meta.sort | dump | safe }}
+            sort: {{ meta.sort | dump | safe }},
+            lang: {{ copy | dump | safe }}
```

Comment:
Open to suggestions on other ways to approach this, but generally it seems to work reasonably well here.

Reply:
I think this is pretty sensible for now. Longer term I was starting to look at https://www.i18next.com/ which looks to be a more full featured i18n library that supports hybrid server/client stuff. Only mentioning in case we do more of this. For now this totally works.